### PR TITLE
Add Chinese translation for index docs

### PR DESCRIPTION
**Pull request by waynerv**
We (a small team) are working to complete the doc translation of Chinese version in the next two months.

### CODE REVIEW

**Comment by RunningIkkyu** [comment]
@waynerv I'm working on the Chinese trans too, may I have your wechat or email?

File: docs/zh/docs/index.md [hunk 1]
```diff
@@ -0,0 +1,438 @@
...
+
+你会看到另一个自动生成的文档 （由 <a href="https://github.com/Rebilly/ReDoc" class="external-link" target="_blank">ReDoc</a>）生成：
+
+![ReDoc](https://fastapi.tiangolo.com/img/index/index-02-redoc-simple.png)
+
+## 升级示例
+
+修改 `main.py` 文件来从 `PUT` 请求中接收请求体。
+
+我们借助 Pydantic 来使用标准的 Python 类型声明请求体。
```
Comment (RunningIkkyu) [inline docs/zh/docs/index.md]: 1. "Standard Python types"  is a proper noun, should be tranlated into `Python标准类型` not `标准的Python类型` .
2.  The relationship between `Pydantic` and `Standard Python types` should be parallel, not progressive。That could lead to ambiguity.

The translation should be 
```txt
我们借助 `Pydantic` 以及 `Python` 标准类型来声明请求体。
```

File: docs/zh/docs/index.md [hunk 2]
```diff
@@ -0,0 +1,438 @@
...
+
+或者一个更复杂的 `Item` 模型：
+
+```Python
+item: Item
+```
+
+......在进行一次声明之后，你将获得：
+
+* 编辑器支持， 包括：
```
Comment (RunningIkkyu) [inline docs/zh/docs/index.md]: Redundant space after "编辑器支持，".

File: docs/zh/docs/index.md [hunk 3]
```diff
@@ -267,175 +263,176 @@ def update_item(item_id: int, item: Item):
...
+
+## 性能
 
-## Performance
 
-Independent TechEmpower benchmarks show **FastAPI** applications running under Uvicorn as <a href="https://www.techempower.com/benchmarks/#section=test&runid=7464e520-0dc2-473d-bd34-dbdfd7e85911&hw=ph&test=query&l=zijzen-7" class="external-link" target="_blank">one of the fastest Python frameworks available</a>, only below Starlette and Uvicorn themselves (used internally by FastAPI). (*)
+独立机构 TechEmpower 所作的基准测试结果显示，基于 Uvicorn 运行的 **FastAPI** 程序是 <a href="https://www.techempower.com/benchmarks/#section=test&runid=7464e520-0dc2-473d-bd34-dbdfd7e85911&hw=ph&test=query&l=zijzen-7" class="external-link" target="_blank">最快的 Python web 框架之一</a>， 仅次于 Starlette 和 Uvicorn 本身（FastAPI 内部使用了它们）。(*)
 
-To understand more about it, see the section <a href="https://fastapi.tiangolo.com/benchmarks/" class="internal-link" target="_blank">Benchmarks</a>.
+要了解更多，请查阅 <a href="https://fastapi.tiangolo.com/benchmarks/" class="internal-link" target="_blank">基准测试</a>章节。
```
Comment (RunningIkkyu) [inline docs/zh/docs/index.md]: Missing a space after "基准测试".

**Review by RunningIkkyu — CHANGES_REQUESTED** [review]
Great translations! @waynerv

**Comment by waynerv** [comment]
@RunningIkkyu Great! You can send your wechat to my gmail: ampedee@gmail.com. I will contact you later.

File: docs/zh/docs/index.md [hunk 4]
```diff
@@ -0,0 +1,438 @@
...
+
+访问 <a href="http://127.0.0.1:8000/redoc" class="external-link" target="_blank">http://127.0.0.1:8000/redoc</a>。
+
+* 备选文档同样会体现新加入的请求参数和请求体：
+
+![ReDoc](https://fastapi.tiangolo.com/img/index/index-06-redoc-02.png)
+
+### 回顾
+
+简单来说， 你声明了 **一次** 参数、请求体等的类型。就和声明函数的参数类型一样。
```
Comment (Dustyposa) [inline docs/zh/docs/index.md]: I can’t make sure this sentence which  be translated is clear.
“总之，你只需要声明参数，请求体等类型一次。和函数参数声明类型相同。” this one is better? need other review

File: docs/zh/docs/index.md [hunk 5]
```diff
@@ -0,0 +1,438 @@
...
+![editor support](https://fastapi.tiangolo.com/img/vscode-completion.png)
+
+<a href="https://fastapi.tiangolo.com/tutorial/">教程 - 用户指南</a> 中有包含更多特性的更完整示例。
+
+**剧透警告**： 教程 - 用户指南中的内容有：
+
+* 对来自不同地方的参数进行声明，如：**请求头**、**cookies**、**form表单** 以及 **上传的文件**。
+* 如何设置 **校验约束** 如 `maximum_length` 或者 `regex`。
+* 一个强大并易于使用的 **<abbr title="也被称为 components, resources, providers, services, injectables">依赖注入</abbr>** 系统。
+* 安全性和身份验证，包括通过 **JWT 令牌** 和 **HTTP Basic** 身份认证来支持 **OAuth2**。
```
Comment (Dustyposa) [inline docs/zh/docs/index.md]: `HTTP Basic auth` is also called "HTTP 基本认证" in [wiki](https://zh.wikipedia.org/wiki/HTTP%E5%9F%BA%E6%9C%AC%E8%AE%A4%E8%AF%81).
It's can be translated.

File: docs/zh/docs/index.md [hunk 6]
```diff
@@ -0,0 +1,438 @@
...
+
+用于 Pydantic：
+
+* <a href="https://github.com/esnme/ultrajson" target="_blank"><code>ujson</code></a> - 更快的 JSON <abbr title="将来自 HTTP 请求中的字符串转换为 Python 数据类型">"解析"</abbr>。
+* <a href="https://github.com/JoshData/python-email-validator" target="_blank"><code>email_validator</code></a> - 用于 email 校验。
+
+用于 Starlette：
+
+* <a href="http://docs.python-requests.org" target="_blank"><code>requests</code></a> - 使用 `TestClient`时安装。
+* <a href="https://github.com/Tinche/aiofiles" target="_blank"><code>aiofiles</code></a> - 使用 `FileResponse` 或 `StaticFiles`时安装。
```
Comment (Dustyposa) [inline docs/zh/docs/index.md]: MIss space before “时安装”

**Comment by jamesyangget** [comment]
天哪，有翻译的人了，我也自己开了个[翻译项目](https://github.com/apachecn/fastapi-docs-cn)，我感觉自己在做无用功

**Comment by Dustyposa** [comment]
> 天哪，有翻译的人了，我也自己开了个[翻译项目](https://github.com/apachecn/fastapi-docs-cn)，我感觉自己在做无用功

has interested in join our team? my email zmzposa@outlook.com

**Comment by tiangolo** [comment]
This is amazing! Thanks everyone for doing this! :rocket: :earth_asia: :speech_balloon: 

And special thanks for the reviews from others, that's key here, as I don't know Chinese :sweat_smile: 

When others feel like this is complete, that there are no more corrections necessary, and hopefully that the style of "conversation" is similar, please add a new review approving the changes of this PR.

That way, after seeing that there's some consensus on the translation I'll be able to merge it. :tada:

**Review by RunningIkkyu — APPROVED** [review]

**Comment by tiangolo** [comment]
@Dustyposa do you think this is ready? You can let me know with a review approving it :smile: :+1: 

@phy25 if you have the chance, could you help me with a review here?

I'm just waiting for more reviews approving it before merging. :globe_with_meridians: :rocket:

**Comment by waynerv** [comment]
@tiangolo I can ask the Chinese community for some help. More and more Chinese developers have started to use FastAPI recently.

**Review by Dustyposa — APPROVED** [review]

**Review by lpdswing — APPROVED** [review]
checked, i approve this

**Review by RunningIkkyu — APPROVED** [review]

**Comment by tiangolo** [comment]
This is awesome! :rocket: :earth_asia: :bow: 

Thanks for all the work @waynerv ! :cake: :clap: :tada: 

Thanks for the reviews and comments @RunningIkkyu , @Dustyposa , @lpdswing , @jamesyangget :muscle: :nerd_face: :bow: :cake:

**Pull request merged**